test-utils/src/test/kotlin/com/google/devtools/ksp/test/KSPAATest.kt

```diff
@@ -265,6 +265,12 @@ class KSPAATest : AbstractKSPAATest() {
         runTest("../kotlin-analysis-api/testData/getPackage.kt")
     }
 
+    @TestMetadata("getSubPackages.kt")
+    @Test
+    fun testGetSubPackages() {
+        runTest("../kotlin-analysis-api/testData/getSubPackages.kt")
```

Comment:
Please replace with: `runTest("../test-utils/testData/api/getSubPackages.kt")`. I.e., don't use the forked version.